runners/katacoda/templates/cloneRepository.md

```diff
@@ -0,0 +1,15 @@
+<%= text; %>
+
+<%= cdCommand; %>
+
+If the parent directories aren't already in the project, 'mkdir -p' will create them for you. 
+
+`mkdir -p <%= directoryPath; %>`{{execute}}
+
+`cd  <%= directoryPath; %>`{{execute}}
```

Comment:
Note the case that the first parameter is an empty string. In this case you do not need these two commands